src/ensembl/src/content/app/browser/browser-cog/BrowserCog.tsx

```diff
@@ -1,8 +1,10 @@
 import React, { useCallback, CSSProperties, useState, useEffect } from 'react';
 import analyticsTracking from 'src/services/analytics-service';
 
-import cogOnIcon from 'static/img/shared/cog-on.svg';
-import cogOffIcon from 'static/img/shared/cog.svg';
+import ImageButton from 'src/shared/components/image-button/ImageButton';
+
+import { Status } from 'src/shared/types/status';
+import { ReactComponent as cogIcon } from 'static/img/shared/cog.svg';
 import BrowserTrackConfig from '../browser-track-config/BrowserTrackConfig';
 import { useTransition, animated } from 'react-spring';
```

Comment:
stylistic remark: can we change the order of imports please (npm modules > helpers/services/actions/etc > components > images > types > styles)?

```
import React, { useCallback, CSSProperties, useState, useEffect } from 'react';
import { useTransition, animated } from 'react-spring';

import analyticsTracking from 'src/services/analytics-service';
import BrowserTrackConfig from '../browser-track-config/BrowserTrackConfig';

import ImageButton from 'src/shared/components/image-button/ImageButton';

import { ReactComponent as cogIcon } from 'static/img/shared/cog.svg';

import { Status } from 'src/shared/types/status';
```

Also, I suspect that we do not need to import or declare the `CSSProperties` type.